client/gutenberg/extensions/simple-payments/edit.js

```diff
@@ -443,30 +459,40 @@ class SimplePaymentsEdit extends Component {
 						/>
 					</div>
 
-					<TextControl
-						aria-describedby={ `${ instanceId }-email-${ fieldEmailError ? 'error' : 'help' }` }
-						className={ classNames( 'simple-payments__field', 'simple-payments__field-email', {
-							'simple-payments__field-has-error': fieldEmailError,
-						} ) }
-						disabled={ isLoadingInitial }
-						label={ __( 'Email' ) }
-						onChange={ this.handleEmailChange }
-						placeholder={ __( 'Email' ) }
-						required
-						type="email"
-						value={ email }
-					/>
-					<HelpMessage id={ `${ instanceId }-email-error` } isError>
-						{ fieldEmailError }
-					</HelpMessage>
-					<HelpMessage id={ `${ instanceId }-email-help` }>
-						{ __(
-							'Enter the email address associated with your PayPal account. Don’t have an account?'
-						) + ' ' }
-						<ExternalLink href="https://www.paypal.com/">
-							{ __( 'Create one on PayPal' ) }
-						</ExternalLink>
-					</HelpMessage>
+					<Form>
+						<BaseControl
+							label={ __( 'Email' ) }
+							id={ `${ instanceId }-email` }
+							className={ classNames( 'simple-payments__field', 'simple-payments__field-email', {
+								'simple-payments__field-has-error': errors.email && touched.email,
+							} ) }
+						>
+							<Field
+								disabled={ isLoadingInitial || isSubmitting }
+								id={ `${ instanceId }-email` }
+								name="email"
+								onBlur={ handleBlur }
+								onChange={ this.handleEmailChange }
+								placeholder={ __( 'Email' ) }
+								required
+								type="email"
+								value={ values.email }
+							/>
+						</BaseControl>
```

Comment:
I'm looking into passing `<TextControl/>` directly to `<Field>` instead of wrapping this to `BaseControl`